packages/@internationalized/date/src/queries.ts

```diff
@@ -238,7 +238,7 @@ export function getWeeksInMonth(date: DateValue, locale: string): number {
 }
 
 /** Returns the lesser of the two provider dates. */
-export function minDate<A extends DateValue, B extends DateValue>(a: A, b: B): A | B {
+export function minDate<A extends DateValue, B extends DateValue>(a?: A, b?: B): A | B | undefined {
```

Comment:
Where do we pass in null/undefined for these?

If we keep this, I guess the parameters shouldn't be optional but should allow null/undefined? I think making them optional only allows undefined not null.

Reply:
https://github.com/adobe/react-spectrum/blob/1cacbf1d438675feb3859fee54b17e620b458d9c/packages/%40react-stately/calendar/src/useRangeCalendarState.ts#L66

Reply:
why allow null? we only pass it undefined

Reply:
Seems like if we are going to allow undefined we'd also want to allow null 🤷

Reply:
I think better not to have to account for both and we just restrict it to one of them. You can check it out in the latest commit though.

Reply:
hmm well the code does account for both already, and did allow null without ts strict, so I was thinking the types should reflect the reality of the implementation.

Reply:
that's fine, it can stay